VultisigApp/VultisigApp.xcodeproj/project.pbxproj

```diff
@@ -4584,7 +4608,7 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/vultisig/commondata";
 			requirement = {
-				branch = main;
+				branch = "feature/message-to-sign";
```

Comment:
can you please get https://github.com/vultisig/commondata/pull/20 merged , and then update the dependencies